packages/jest-snapshot/src/inline_snapshots.ts

```diff
@@ -8,50 +8,60 @@
 import * as fs from 'fs';
 import * as path from 'path';
 import semver = require('semver');
-import {
-  CallExpression,
-  file,
-  templateElement,
-  templateLiteral,
-} from '@babel/types';
-import type {Frame} from 'jest-message-util';
+import type {PluginItem} from '@babel/core';
+import type traverse from '@babel/traverse';
+import type {CallExpression, Expression, File, Program} from '@babel/types';
 
 import type {Config} from '@jest/types';
+import type {Frame} from 'jest-message-util';
 import {escapeBacktickString} from './utils';
 
+// TODO ignore this mess - with a Babel plugin, these will be MUCH, MUCH nicer for TS
```

Comment:
Just curious since this is (/was) my PR originally (by the way thanks so much for jumping on it! I was stuck with the jest circus failures, so I really appreciate it). Is this going to go in like this and get cleaned up later, or is the Todo for this PR? If it'll be around for a little while, it might be easier to understand if there was a helper function

```typescript
const vmRequire = (module name: string) => {
  return require(require.resolve(module name), {
    [Symbol.for('OUTSIDE_JEST_VM_RESOLVE_OPTION')]: true
  }))
}
```
or similar.

That way the function could have some jsdoc explanation indicating why it's needed.

Reply:
Hi, we will land some other PRs concerning the mechanism this uses first - after that, this will be changed to look pretty much like a normal `require`